Simulations/EUREC4A-MIP/control/setup/bc.yaml

```diff
@@ -0,0 +1,27 @@
+plugins:
+  source:
+    - module: intake_xarray
+
+sources:
+
+  ERA5:
+    args:
+      urlpath: simplecache::swift://swift.dkrz.de/dkrz_0913c8f3-e7b6-4f94-9221-06880d4ccfea/CTRL_ERA5/cas{{date.strftime("%Y%m%d%H%M%S")}}.nc
```

Comment:
During the last years, it came more and more apparent, that large datasets are much easier to handle than small datasets (because cutting them apart is much easier than glueing them together). Especially, as the example in the pull request shows that the intended use is "glueing the things together", I'd ask if it would be more reasonable to make the data available as one dataset. In this case, the example would boild down to e.g.:

```python
import xarray as xr
from intake import open_catalog

cat = open_catalog("catalog.yml")
ds = cat.simulations.EUREC4A_MIP.control.setup.boundary_conditions.to_dask() \
    .sel(time=slice('2020-01-01','2020-02-01'))  # control
#ds.to_netcdf("EUREC4A-MIP_boundaryConditions.nc")
```

I could see two ways to go there:
* keep the netCDF files as they are and add a kerchunk reference file to access the dataset
* concatenate and convert the netCDF files to zarr and re-upload them

Reply:
Yeah, I'm totally with you here @d70-t. Unfortunately, I am not the creator of the dataset and I do not want to post-process the dataset myself. This dataset is however a bit special in the sense that it will be input to climate simulations and they require mostly netCDF files which might even be allowed to only contain individual days or time steps.

That being said, I like the idea of using kerchunk references here as it would solve this dilemma. This could also be a good idea for some other datasets, e.g. the cloud radar data from the Meteor. Where would be a good place to store those index files?

Reply:
The reference files for this case will be saved alongside the output files.